routers/users.py

```
@@ -0,0 +1,257 @@
+        update_data : Annotated[UserUpdate, Depends(UserUpdate.as_form)],
+        current_user:  Annotated[dict, Depends(get_current_user)]):
+    users = get_db_users()
+    user =next((i for i,u in enumerate(users) if u["user_id"] == current_user["user_id"]),None)
+    if user is None:
+        raise HTTPException(status_code=404,detail="해당 유저가 없습니다")
+
+    patch_dict = update_data.model_dump(exclude_unset=True,mode="json")
+
+    users[user].update(patch_dict)
+#왜 커밋 안되?
+    with open(DB_FILE,"w",encoding="utf-8") as f:
```

Ruff (0.14.11)

Async functions should not open files with blocking methods like `open`

(ASYNC230)